scout/server/blueprints/institutes/controllers.py

```diff
@@ -460,30 +460,7 @@ def gene_variants(store, pymongo_cursor, variant_count, page=1, per_page=50):
             continue
 
         genome_build = get_genome_build(variant_case_obj)
-        variant_genes = variant_obj.get("genes")
```

Comment:
I've simply moved this code to another function, since sonarcloud was complaining for code complexity